Sources/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift

```diff
@@ -2,6 +2,8 @@ import UIKit
 
 enum MiniAppJSActionCommand: String {
     case getUniqueId
+    case getContactId
+    case getMauid
```

Comment:
Thanks for adding here, `getContactId` name is certainly ambiguous

Reply:
yeah I've replaced it according to the js sdk
uniqueId is still available but contactId is now messageUniqueId